stdlib/public/TensorFlow/NumpyConversion.swift

```diff
@@ -87,6 +93,12 @@ extension ShapedArray : ConvertibleFromNumpyArray
 
 extension Tensor : ConvertibleFromNumpyArray
   where Scalar : NumpyScalarCompatible {
+  /// Creates a tensor with the same shape and scalars as the specified
+  /// `numpy.ndarray` instance.
+  ///
+  /// - Parameter numpyArray: The `numpy.ndarray` instance to convert.
+  /// - Precondition: The `numpy` Python package must be installed.
+  /// - Precondition: `numpyArray` must have a compatible scalar datatype.
```

Comment:
"datatype" isn't a thing in either Numpy or Swift. `dtype` (backquoted) is probably more appropriate.

```suggestion
  /// - Precondition: `numpyArray` must have a compatible scalar datatype.
```

Reply:
Thanks! Done in https://github.com/apple/swift/pull/23069/commits/e7920e443eeb844ac662c91069d9c9025a3c2439.